### PR TITLE
BlockQuote, Youtube component and code refactor for sass files

### PR DESCRIPTION
**Pull request by nickhsine**
### spec is here: 
- screen shot 2016-07-12 at 5 33 43 pm
- 

@garfieldduck

### CODE REVIEW

**Comment by hcchien** [comment]
LGTM besides the style

File: src/themes/pre-bootstrap-customization.scss [hunk 1]
```diff
@@ -1,48 +1,5 @@
...
-
-//** Computed "line-height" (`font-size` * `line-height`) for use with `margin`, `padding`, etc.
-$line-height-computed:    floor(($font-size-base * $line-height-base)) !default; // ~30px
-
-//** By default, this inherits from the `<body>`.
-$headings-font-family:    $font-family-base !default;
-$headings-font-weight:    400 !default;
-$headings-line-height:    1.1 !default;
-$headings-color:          inherit !default;
+$font-family-sans-serif: "Noto Sans",'Noto Sans TC', "Apple LiGothic Medium", Roboto, "Microsoft JhengHei", "Microsoft JhengHei", "新細明體", "Lucida Grande", "Lucida Sans Unicode", sans-serif;
```
Comment (garfieldduck) [inline src/themes/pre-bootstrap-customization.scss]: I found maybe we should change it as:

"Noto Sans TC", "Noto Sans", "PingFang TC", "Apple LiGothic Medium", Roboto, "Microsoft JhengHei", "Lucida Grande", "Lucida Sans Unicode", sans-serif;

File: src/components/article/HeadingAuthor.js [hunk 2]
```diff
@@ -11,7 +13,7 @@ export class HeadingAuthor extends Component {
 
   _formatAuthor(authors) {
     const authorTypes = {
```
Comment (garfieldduck) [inline src/components/article/HeadingAuthor.js]: Take it into outer config (?

**Comment by garfieldduck** [comment]
LGTM :+1:  :sunny:  :shipit:

**Comment by nickhsine** [comment]
@garfieldduck 再幫忙看一下，根據你的建議來修改。

**Comment by garfieldduck** [comment]
LGTM :+1:  :+1:  :+1:

**Pull request merged**